README.md

```diff
@@ -1402,6 +1438,51 @@ Type: `string`
     ad_integration_dns_connection_type: ethernet
 ```
 
+##### Configuring AD integration without joining to AD
+
+You must join managed host to AD Server yourself prior to running this playbook.
+
+```yaml
+- name: Configure with AD server authentication without joining to AD
+  hosts: all
+  vars:
+    mssql_accept_microsoft_odbc_driver_17_for_sql_server_eula: true
+    mssql_accept_microsoft_cli_utilities_for_sql_server_eula: true
+    mssql_accept_microsoft_sql_server_standard_eula: true
+    mssql_version: 2022
+    mssql_password: "p@55w0rD"
+    mssql_edition: Evaluation
+    mssql_manage_firewall: true
+    mssql_ad_configure: true
+    mssql_ad_join: false
+    mssql_ad_sql_user_name: sqluser
+    ad_integration_realm: domain.com
+    ad_integration_user: Administrator
+    ad_integration_password: Secret123
+```
+
+##### Configuring AD integration without joining to AD and without obtaining Kerberos ticket
+
+You must join managed host to AD Server yourself prior to running this playbook.
+
+```yaml
+- name: Configure with AD server authentication without joining to AD
+  hosts: all
+  vars:
+    mssql_accept_microsoft_odbc_driver_17_for_sql_server_eula: true
+    mssql_accept_microsoft_cli_utilities_for_sql_server_eula: true
+    mssql_accept_microsoft_sql_server_standard_eula: true
+    mssql_version: 2022
+    mssql_password: "p@55w0rD"
+    mssql_edition: Evaluation
+    mssql_manage_firewall: true
+    mssql_ad_configure: true
+    mssql_ad_join: false
+    mssql_ad_obtain_kerberos: false
+    mssql_ad_sql_user_name: sqluser
```

Comment:
Is it worth renaming `mssql_ad_sql_user_name` to `mssql_ad_sql_user` to match `ad_integration_user`?

Reply:
`mssql_ad_sql_user_name` is already part of the public API - we can't just get rid of it without breaking user playbooks

Reply:
Ok. It was just a suggestion in order to standardize variable names.

As a way to allow renaming a variable, you could define it as:
```yaml
mssql_ad_sql_user: "{{ mssql_ad_sql_user_name | default('sqluser') }}"
```

Reply:
Yeah, I can deprecate mssql_ad_sql_user_name and use mssql_ad_sql_user instead. I'll do it in a separate PR after the on-going PRs are merged.

Reply:
Done in https://github.com/linux-system-roles/mssql/pull/220